### DISCUSSION TITLE
[req] Correctly format the locations of all the sections

### DISCUSSION THREAD

**Discussion by NicTanghe** [chuangcaleb/obsidian-fountain-editor#36] [Ideas]
Place scene numbering on the right hand side
Below is the css i used to use.
```css
<div style="text-align: center;">
   <div style="display: flex; justify-content: space-between;">
      <span style="font-weight: bold; font-family: 'Courier New', monospace;">
      scene
      </span>
      <span style="font-weight: bold; font-family: 'Courier New', monospace;">
      1
      </span>
   </div>
</div>
```

**Comment by chuangcaleb** [comment]
Hi @NicTanghe. The intention waas that I didn't want to bloat the Issues tab with many entries, but I thought it would be better to flesh them out on the discussions page before creating feature requests in Issues. I know there are different ways how people administrate, but feel free to evaluate the strengths and weaknesses of different strategies, we can adapt!

---

Regarding your Idea, could you elaborate further? is this a brand new feature request, or improving an existing one? Seems to me like you want scene numbering.

**Reply by NicTanghe** [reply]
------
If I`m not mistaken at this moment in time there is no css ects applied to the page numbering as it remains right next to the locations.
It whould be nice if the page numbering whould also be placed on the right hand side next to the location.

(i dont mean automatic scene numbering ) i just mean having the scene numbering be placed in the correct location.

**Reply by chuangcaleb** [reply]
sorry, I genuinely missed this notification. I understand the intended page numbering behaviour but I don't think scene numbering is something that Fountain was made to handle. Are you manually adding the scene numbers? I'm going to need more context on the whole intended workflow here

**Reply by NicTanghe** [reply]
Hej np. glad you made this in the first place.
Im somewhat trying to make an editor for markdown in rust and also have not added this becouse of pother priorities.

Here`s the docs.

Power user: Scene Headings can optionally be appended with Scene Numbers. Scene numbers are any alphanumerics (plus dashes and periods), wrapped in #. All of the following are valid scene numbers:

INT. HOUSE - DAY #1#
INT. HOUSE - DAY #1A#
INT. HOUSE - DAY #1a#
INT. HOUSE - DAY #A1#
INT. HOUSE - DAY #I-1-A#
INT. HOUSE - DAY #1.#
INT. HOUSE - DAY - FLASHBACK (1944) #110A#
https://fountain.io/syntax/

**Reply by chuangcaleb** [reply]
10 months later and he's still here T.T Thanks for sticking around!

Thank you for added context of the Scene Numbers from the Fountain Syntax docs. I don't use them, so I completely forgot they were a thing.

I'm using BetterFountain for compiling, so the scene numbering syntax is rendered like this:

<img width="1440" alt="44ffb2cdf39c134e84ecdc5e04485308ddd40cb5701cf11c8aa28a6468be8ece" src="https://github.com/user-attachments/assets/2e0ddff6-acc3-4579-bd2a-aa7f02086272" />

The thing is that Fountain is an authoring spec, and doesn't specify how to display/render scene numbers. Was something like this what you had in mind? Also, I'm trying to clean up the code, you can check out the dev branch, so priority on this isn't too high, since the raw markup doesn't look too bad imo

<img width="606" alt="adfcebf82a3933048867a5f82ca7d3e272b1a10498657f02a8b6e4856299e515" src="https://github.com/user-attachments/assets/31bf854d-7171-47af-8594-7ff8313ecfb8" />

**Reply by NicTanghe** [reply]
Just the same as better fountain i gues ?
